cpp/src/scalar/scalar_factories.cpp

```diff
@@ -125,7 +125,7 @@ template <>
 std::unique_ptr<cudf::scalar> default_scalar_functor::operator()<list_view>(
   rmm::cuda_stream_view stream, rmm::mr::device_memory_resource* mr)
 {
-  CUDF_FAIL("list_view type not supported");
+  return std::make_unique<list_scalar>();
```

Comment:
This is not right. The `stream` and `mr` is not being passed to construct the validity data of `list_scalar`.